SPECS/binutils/binutils.spec

```diff
@@ -26,7 +29,9 @@ It contains the libraries and header files to create applications
 for handling compiled objects.
 
 %prep
-%autosetup -p1
+%setup -q 
```

Comment:
You should be able to leave this as "%autosetup -p1" and it will apply all of the patches. Any reason to change this?